pkg/model/utils.go

```diff
@@ -17,22 +17,70 @@ package model
 
 import (
 	"github.com/jhump/protoreflect/dynamic"
+	"log"
+	"os"
 )
 
+var error = log.New(os.Stderr, "ERROR: ", 0)
+
 func GetEnumValue(val *dynamic.Message, name string) string {
-	return val.FindFieldDescriptorByName(name).GetEnumType().
-		FindValueByNumber(val.GetFieldByName(name).(int32)).GetName()
+	fd := val.FindFieldDescriptorByName(name)
+	if fd == nil {
+		error.Fatalf("fieldDescriptor is nil for : %s", name)
```

Comment:
Should not be generating a fatal error; instead return a error and let the caller decide how it wants to handle the error. Introduce error as second return value of this method.

Same for next 2 methods.